packages/cli-e2e/utils/cli.ts

```diff
@@ -76,8 +75,7 @@ export function setupUIProject(
 
 export function getConfig() {
   const pathToConfig = resolve(
-    __dirname,
-    ...new Array(4).fill('..'),
+    homedir(),
```

Comment:
IMO, if you go 'beyond' the root folder with `..` that's risky. so let's use homedir instead.